### ISSUE TITLE
inline.preboot.code.ts throws warning with Webpack

### ISSUE THREAD

**Issue by mogusbi** [angular/preboot#38]
**Note: for support questions, please use the Universal Slack Channcel or https://gitter.im/angular/universal** 

* **I'm submitting a ...**
- [x] bug report
- [ ] feature request

* **Which parts of preboot are affected by this issue?**
- [ ] server side
- [ ] client side
- [ ] inline
- [x] build process
- [ ] docs
- [ ] tests

* **Do you want to request a *feature* or report a *bug*?**
Bug

* **What is the current behavior?**
When building the server side application Webpack throws this warning

```
WARNING in /Volumes/Data/Users/mo/Repos/universal-test/~/preboot/src/server/inline.preboot.code.ts
41:12-40 "export 'hasOwnProperty' (imported as 'eventRecorder') was not found in './event.recorder'
```

* **What is the expected behavior?**
No warnings to be thrown

* **Please tell us about your environment:**

- Browser: n/a
- Language: TypeScript 2.4.2
- OS:  Mac OS X
- Platform: NodeJs

**Comment by jeffwhelpley** [comment]
This should be fixed with `5.1.4`.

**Comment by kamalakkanni** [comment]
hi,this is not fixed with 5.1.4
